src/query/typed_value.hpp

```diff
@@ -420,6 +421,9 @@ class TypedValue {
     new (&graph_v) std::unique_ptr<Graph>(graph_ptr);
   }
 
+  explicit TypedValue(std::function<void(TypedValue *)> &&other)
```

Comment:
I would pass it as value and then move it not necessarily as rvalue reference.

Reply:
But this is move ctor, so I think that should be fine